brave/brave-ios
# Fix #1661, Fix #1662: Show graphic & icons when disabling Rewards AC/Ads

Also updates Auto-Contribute copy to match desktop

@jamesmudgett (or @jenn-rhim) to verify design

## Summary of Changes

This pull request fixes issue #1661 
This pull request fixes issue #1662 

## Submitter Checklist:


## Test Plan:

- Enable rewards
- In rewards settings, turn off A/C & Ads
- Verify graphics + text appear in place of the view details button
- Change region to unsupported region
- Verify disabled graphic + text _does not_ appear for ads while it is unsupported

## Screenshots:

| Disabled | Unsupported |
| --- | --- |
| ![Simulator Screen Shot - iPhone 11 Pro - 2019-11-28 at 17 31 41](https://user-images.githubusercontent.com/529104/69833973-5b0df800-1205-11ea-9887-0614ddc571e6.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2019-11-28 at 17 31 01](https://user-images.githubusercontent.com/529104/69833976-5e08e880-1205-11ea-8d91-ba5a5ab60d67.png) |

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
